syz-manager/rpc.go

```diff
@@ -548,6 +560,9 @@ func (serv *RPCServer) createExecOpts(req *fuzzer.Request) ipc.ExecOpts {
 }
 
 func (runner *Runner) logProgram(procID int, p *prog.Prog) {
+	if runner.stopFuzzing.Load() {
```

Comment:
For consistency it's better to move this to StartExecuting.
But even better to findRunner.